### DISCUSSION TITLE
How to change background/text colours?

### DISCUSSION THREAD

**Discussion by HubKing** [webmin/authentic-theme#1628] [Q&A] [answered]
I installed webmin, and it seems by default it comes with two themes: Authentic (dark) and Gray framed (light). My problem is that the contrast of the dark theme is too low. That is, the background is too bright and the text is too dark. I want to make the background black(#000000), and the text white (#FFFFFF). Can I do that?

I have visited the theme configuration page, but it only allowed to set images for background or logos, etc, not change colours.

![image](https://user-images.githubusercontent.com/10900489/195980743-d69353cf-4142-4e2c-84b0-8c28a45b1dc0.png)

<html><body>

Authentic theme version | 20.01.1:5
-- | --

</body>
</html>

**Comment by iliaross** [comment, marked as answer]
>  My problem is that the contrast of the dark theme is too low

Why not using Authentic Theme in the light mode instead? You can switch dark/light modes by clicking `Alt+L` or using moon/sun icon in navigation menu.

> I want to make the background black(#000000), and the text white (#FFFFFF). Can I do that?

Technically, yes you could look at `nightrider.min.css` file and apply changes using `Theme Configuration: General defaults -> Theme Extensions` page.

**Reply by HubKing** [reply]
Light mode scorches my eyes. 

I don't know where nightrider.min.css is, and I am not good at CSS. I do not think most users are good at CSS. It would be a lot more user-friendly to have GUI for changing basic colours.

**Reply by iliaross** [reply]
I will add to my todo to further improve dark mode.

**Reply by HubKing** [reply]
> I will add to my todo to further improve dark mode.

Thanks. I guess maybe instead of letting the user customise/modifying existing themes, just having a separate high contrast dark theme (that is, in addition to existing two themes) would be the easiest for users. A few IDE's or apps I know do this. For example, Visual Studio Code has "Dark High Contrast", and so does Android Studio (all JetBrains IDE's).
